beta/src/components/Layout/Nav/Nav.tsx

```diff
@@ -101,11 +101,18 @@ export default function Nav({
 }) {
   const [isOpen, setIsOpen] = useState(false);
   const [showFeedback, setShowFeedback] = useState(false);
+  const [isFirefox, setIsFirefox] = useState(false);
   const scrollParentRef = useRef<HTMLDivElement>(null);
   const feedbackAutohideRef = useRef<any>(null);
   const {asPath} = useRouter();
   const feedbackPopupRef = useRef<null | HTMLDivElement>(null);
 
+  useEffect(() => {
+    if (typeof window !== undefined) {
+      setIsFirefox(window.navigator.userAgent.indexOf('Firefox') > -1);
```

Comment:
if you do this in an effect, the issue would still exist until hydration. then, by setting state, it would cause an extra re-render of the nav. seems not ideal. we should fix the CSS ideally.